contents/blog/best-open-source-session-replay-tools.md

```diff
@@ -0,0 +1,195 @@
+---
+date: 2022-04-07
+title: Open source (and self-hosted) alternatives to Hotjar, FullStory & LogRocket
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-company-culture-blog.png
+featuredImageType: full
+author: ["hanna-crombie"]
+categories: ["Guides", "Product analytics"]
+---
+
+Most analytics tools have one goal in mind: providing actionable insights on user behavior. It's all about hard numbers; views, clicks, conversions. But sometimes all those numbers can feel a bit sterile. Separated from the individual user narrative, statistics are an incomplete picture. Enter session recording.
+
+### Why you need session recording
+
+Session recordings, also known as session replays, are video renderings of actions taken by your users in real-time. They capture mouse movement, clicks, taps, and scrolling across your site or app’s pages.
+
+Aside from making for some fascinating watching, recordings facilitate a detailed investigation of user behavior, allowing you to identify friction areas such as: 
+
+- Where users are getting stuck on your site
+- Hesitation at different stages of your funnel
+- How bugs come about
+- How your users are interacting with individual elements of your product.
+
+This context can help you to optimize user experience, build user-led products and, ultimately, improve conversion and retention. 
+
+Bottom line: session recording is an essential tool for making better products.
```

Comment:
```suggestion
> Bottom line: session recording is an essential tool for making better products.
```

Reply:
Kind of feel like a boxout would look weird here... maybe we bold up 'Bottom line:' for emphasis?

Reply:
Sure, sounds good